wordpress-mobile/WordPress-Android
# Issue/13326 init scan history

Parent issue #13326 

This PR introduces ScanHistory screen and links to it from a toolbar item on the Scan screen.

The screen doesn't contain a viewpager yet - it's coming in the upcoming PR.

Open questions:
1.Single vs multiple activities - Do you prefer introducing a new ScanHistory activity or re-using ScanActivity and just replace fragments? -> ✅  we decided to use multiple activites

To test:
1. Make sure "Scan" feature flag is enabled
2. Open scan section
3. Click on "History" button in the toolbar
4. Notice the history with "Hello" text is shown


PR submission checklist:
